PDFExporter/Categories/UIKit/UITableView+PDFExporterDrawing.m

```diff
@@ -134,4 +108,27 @@ - (CGRect)subviewRect:(UIView *)subview layoutPageRect:(CGRect)rect {
     }
 }
 
+- (void)setFrameForContentOffset:(CGPoint)contentOffset {
+    PDFExporter_dispatch_sync_main_queue(^{
+        self.contentOffset = contentOffset;
+        CGRect frame = self.frame;
+        [self layoutIfNeeded];
+        self.frame = frame;
+        [self layoutHeadersAndFooters];
+    });
+}
+
+- (void)computeOffsetForRect:(CGRect)rect additionalCode:(void (^)(void))additionalCode {
```

Comment:
The function's name is vague. 
As I see it doesn't compute the contentOffset, it actually 'scrolls' the entire content. Therefore I propose the following name:
```objective-c
- (void)scrollContentForRect:(CGRect)rect usingBlock:(void (NS_NOESCAPE ^)(void))block
```

Reply:
done.